compiler/rustc_middle/src/ty/relate.rs

```diff
@@ -29,7 +29,7 @@ pub trait TypeRelation<'tcx>: Sized {
     fn tag(&self) -> &'static str;
 
     /// Returns whether or not structural equality can be used to fast-path this relation
-    fn fast_equate_combine(&self) -> bool;
+    fn fast_equate(&self) -> bool;
```

Comment:
Annotate inline this declaration instead?

Reply:
`#[inline]` means nothing on trait methods without a body.